src/components/CmsSearch.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+  <!-- Search Bar -->
+  <div class="search-bar">
+    <v-text-field
+      v-model="typedQuery"
+      filled
+      rounded
+      placeholder="Search..."
+      class="search-text"
+      @keydown.enter="updateSearchQuery()"
+    >
+      <template #prepend-inner>
```

Comment:
So I thought we had decided to leave the magnifying glass on the right side of the search bar, so users have something to tap when they are ready to search? And then it would be replaced with the close-icon only when the search-query matches the most recent search (to clear the previous search results)?

If we are moving the search icon, maybe it should still be tappable?

Reply:
talked through this with jacob and carlos. we are going to mimic existing search bars for mobile devices, hence the change